src/Shared/CommunicationsUtilities.cs

```diff
@@ -60,7 +60,12 @@ internal enum HandshakeOptions
         /// <summary>
         /// Building with administrator privileges
         /// </summary>
-        Administrator = 32
+        Administrator = 32,
+
+        /// <summary>
+        /// Using the .NET Core/.NET 5.0+ runtime
+        /// </summary>
+        NET = 64,
```

Comment:
It looks like this comment should be updated:
https://github.com/dotnet/msbuild/blob/37c61113335898d83fcae3203846402f2af31bf1/src/Shared/CommunicationsUtilities.cs#L78